frontend/l10n.yaml

```diff
@@ -1,3 +1,3 @@
 arb-dir: lib/l10n
-template-arb-file: app_en.arb
+template-arb-file: app_de.arb
```

Comment:
why this change?

Reply:
Basically it doesn't really matter as long as the translation files are identical/have the same keys. But since (at least atm) all our apps/white labels have german as a language but not every app has english I figured using german as a default scheme makes more sense.